Userland/Applications/Piano/PlayerWidget.cpp

```diff
@@ -61,3 +98,17 @@ PlayerWidget::PlayerWidget(TrackManager& manager, AudioPlayerLoop& loop)
 PlayerWidget::~PlayerWidget()
 {
 }
+
+void PlayerWidget::add_track()
+{
+    m_track_manager.add_track();
+    auto latest_track_count = m_track_manager.track_count();
+    auto latest_track_string = String::formatted("{}", latest_track_count);
+    m_track_number_choices.append(latest_track_string);
+    m_track_dropdown->set_selected_index(latest_track_count - 1);
+}
+
+void PlayerWidget::next_track()
+{
+    m_track_dropdown->set_selected_index(m_track_manager.next_track_index());
```

Comment:
Why does this method not select the next track in the track manager?

Reply:
There's an on_change listener on the dropdown that will do that (`m_track_manager.set_current_track`), that on_change also works when you select from the drop down. Otherwise I think we'd have to select the track here and update the dropdown without triggering the onChange (or be ok with `set_current_track` being called twice every time the "next_track" was called), I figured this was cleaner.